setup.py

```diff
@@ -39,7 +39,7 @@
     install_requires=['pyephem',
                       'katpoint',
                       'matplotlib<3',
-                      'numpy',
+                      'numpy<1.17',
```

Comment:
This is fine for Python 2, but astrokat can also run under Python 3, and no need to limit  it there.
We can do something like this, to have different packages on different versions:  https://github.com/ska-sa/katportalclient/blob/master/setup.py#L41-L42